src/homeworks/homework15/zooclub/ClubService.java

```diff
@@ -81,17 +83,31 @@ public void showClub() {
                         System.out.print(s.getTypeAnimal() + " " + s.getNickname() + " ");
                     }
                     System.out.println();
-                }
-            }
+                }*/
+            mapPerson.entrySet()
+                    .stream()
+                    .peek(k -> System.out.print(k.getKey().getName() + ": "))
+                    .peek(v -> v.getValue()
```

Comment:
peek() is not used that way, it's better to use map()
Because peek() is for debugging purpose -https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#peek-java.util.function.Consumer-